assets/samplesheet.csv

```diff
@@ -1,3 +1,2 @@
-sample,fastq_1,fastq_2
-SAMPLE_PAIRED_END,/path/to/fastq/files/AEG588A1_S1_L002_R1_001.fastq.gz,/path/to/fastq/files/AEG588A1_S1_L002_R2_001.fastq.gz
-SAMPLE_SINGLE_END,/path/to/fastq/files/AEG588A4_S4_L003_R1_001.fastq.gz,
+sample,file,family_id,paternal_id,maternal_id,sex,phenotype
+HG002_Revio,/media/ssd_4tb/felix/projects/skierfe/assets/test_data/HG002_PacBio_Revio.fastq.gz,FAM1,XXX,YYY,1,1
```

Comment:
I would update this to reflect the location of the test data that you just uploaded

Reply:
Yes! Think I added things correctly now.